Formula/csound.rb

```diff
@@ -0,0 +1,66 @@
+class Csound < Formula
+  desc "Sound and music computing system"
+  homepage "https://csound.com"
+  url "https://github.com/csound/csound/archive/6.12.2.tar.gz"
+  sha256 "39f4872b896eb1cbbf596fcacc0f2122fd3e5ebbb5cec14a81b4207d6b8630ff"
+
+  depends_on "cmake" => :build
+
+  depends_on "fltk"
+  depends_on "liblo"
+  depends_on "libsndfile"
+  depends_on "stk"
+
+  def install
+    inreplace "CMakeLists.txt",
+      %r{^set\(CS_FRAMEWORK_DEST\s+"~/Library/Frameworks"\)$},
+      "set(CS_FRAMEWORK_DEST \"#{frameworks}\")"
+
+    # Csound is not compatible with FluidSynth 2
+    # (https://github.com/csound/csound/issues/1078), which was added in
+    # https://github.com/Homebrew/homebrew-core/commit/85bb1f4c28c5412f97c718db6415077c35481acc.
+    args = std_cmake_args + %W[
+      -DBUILD_FLUID_OPCODES=OFF
+      -DBUILD_JAVA_INTERFACE=OFF
+      -DBUILD_LUA_INTERFACE=OFF
+      -DBUILD_PYTHON_INTERFACE=OFF
+      -DCMAKE_INSTALL_RPATH=#{frameworks}
+    ]
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make", "install"
+
+      include.install_symlink "#{frameworks}/CsoundLib64.framework/Headers" => "csound"
+    end
+  end
+
+  def caveats; <<~EOS
+    You should add to your .bash_profile:
```

Comment:
This does not sound specific to Homebrew, in which case we want to remove the `caveats`

Reply:
I’m happy to remove this, but I think most of the time, most people who use this formula won’t know to update their .bash_profile with Homebrew paths. Also, similar things seem to occur in a few other formulae (but not many):

https://github.com/Homebrew/homebrew-core/blob/1b23cc14a8162dd0bb59977b23a0e00c62941217/Formula/glassfish.rb#L19-L23

https://github.com/Homebrew/homebrew-core/blob/7253b5839475732e96b2dfa3a74703332c7bf34c/Formula/wildfly-as.rb#L18-L25

https://github.com/Homebrew/homebrew-core/blob/8eb7ac11ae4751238b374dbc636e168879bef093/Formula/payara.rb#L25-L30

Reply:
What's the output when you run the software without these variables set ?

Reply:
The effect of not setting these variables is that Csound opcodes that rely on .dylib files in CsoundLib64.framework and [STK (Synthesis ToolKit) opcodes](https://csound.com/docs/manual/STKTop.html) won’t be usable (they’ll result in a syntax error) even though they’re installed.

For example, running the Csound files that are created in the `test do` block without these variables set would fail because of `STKPlucked`, an STK opcode:

```
error: syntax error, unexpected T_IDENT  (token "STKPlucked") from file test.orc (1)
 line 6:
>>>          aSignal STKPlucked <<<
Unexpected untyped word aSignal when expecting a variable
Parsing failed due to invalid input!
Stopping on parser failure
cannot compile orchestra
```

Reply:
I’ve removed the caveats in the interest of getting this merged.